pkgs/by-name/xi/xiasl/package.nix

```
@@ -0,0 +1,80 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeDesktopItem
+, copyDesktopItems
+, qt5
+, libsForQt5
+, imagemagick
+, acpica-tools
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "xiasl";
+  version = "1.1.67";
+
+  src = fetchFromGitHub {
+    owner = "ic005k";
+    repo = "Xiasl";
+    rev = finalAttrs.version;
+    hash = "sha256-6RJeNLgeFchCjYLcwKgGtV0dLyHO7gYpIpo/3BBh/wM=";
+  };
+
+  buildInputs = [
+    qt5.qtbase
```

```suggestion
    libsForQt5.qtbase
```

Also for other `qt5.*` package. `qt5` in the input should be removed as well.

---

Done.